.github/workflows/build.yaml

```diff
@@ -38,12 +38,12 @@ jobs:
         run: |
           app_version=$(yq '.version' rockcraft.yaml)
           version=$(yq '(.version|split("-"))[0]' rockcraft.yaml)
-          base=$(yq '(.base|split(":"))[1]' rockcraft.yaml)
+          base=$(yq '(.base|split("@"))[1]' rockcraft.yaml)
           risk=edge
           tag=${version}-${base}_${risk}
 
-          sed -i "s/${app_version}/${tag}/g" rockcraft.yaml
           rockcraft pack --verbose
+          mv charmed-kafka_${version}_amd64.rock charmed-kafka_${tag}_amd64.rock
```

Comment:
**niptick** maybe we can avoid this renaming, and use `version` also [here](https://github.com/canonical/charmed-kafka-rock/pull/11/files#diff-c0c68244c39d0231fb6c8bf8113bb4673a9f6e833bdbc78f11904647ff40a46bR34) and [here](https://github.com/canonical/charmed-kafka-rock/blob/feat/dpe-2707-add-security-scanner/.github/workflows/publish.yaml#L44)

That would make thinks a bit more consistent